app/routes/individual_response.py

```diff
@@ -152,10 +155,31 @@ def get_individual_response_post_address_confirmation(schema, questionnaire_stor
         list_item_id=None,
     )
 
-    if request.method == "GET":
-        return render_template(
-            template="individual_response/confirmation",
-            display_address=questionnaire_store.metadata.get("display_address"),
-        )
+    if request.method == "POST":
+        return redirect(url_for("questionnaire.get_questionnaire"))
+
+    return render_template(
+        template="individual_response/confirmation",
+        display_address=questionnaire_store.metadata.get("display_address"),
+    )
+
+
+@individual_response_blueprint.route("/who", methods=["GET", "POST"])
```

Comment:
The previous link for this journey should back to the previous page. It should only go to the hub when on you are back on the `IR request launcher` page.

Another two scenarios:
- **1 non-primary person only:**
	- The previous link on the how page is also adding list item id to the url which is taking it back to the individual section from `IR request launcher` page instead of the hub. Regardless of the number of people, the previous should go back to the hub if we know they came from the hub?.

- **at least 2 non-primary person**
	- The previous link from any IR page is going straight to the hub, instead of back sequentially to the `IR request launcher` page then the hub.

Hopefully that is clear, if not, happy to discuss further.

Reply:
I think the card has overstated this somewhat which has made it confusing "Previous link should always go back to hub no matter how far along the IR journey you are (ensure alternate journey goes to individual section introduction page)."

Reply:
I agree that comment has overstated it @iwootten - I think it was added to differentiate between the hub originated journey and the individual section originated journey, but that would be default behaviour as you pointed out 😄